enterprise_access/apps/subsidy_access_policy/mocks.py

```diff
@@ -0,0 +1,81 @@
+""" Mock clients for subsidy_access_policy models. """
+
+import mock
+
+
+class group_client():
+    """
+    API client for the groups service.
+    """
+    @classmethod
+    def group_contains_learner(cls, group_uuid, learner_id):
+        """Group service api"""
+        return mock.MagicMock(group_uuid, learner_id)
+
+    @classmethod
+    def get_groups_for_learner(cls, learner_id):
+        """Group service api"""
+        return mock.MagicMock(learner_id)
+
+class catalog_client():
+    """
+    API client for the enterprise-catalog service.
+    """
+    @classmethod
+    def catalog_contains_content(cls, catalog_uuid, content_key):
+        """Catalog service api"""
+        return mock.MagicMock(catalog_uuid, content_key)
+
+class subsidy_client():
+    """
+    API client for the groups service.
+    """
+    @classmethod
+    def is_redeemable(cls, subsidy_uuid, learner_id, content_key):
+        """Subsidy service api"""
+        return mock.MagicMock(subsidy_uuid, learner_id, content_key)
+
+    @classmethod
+    def redeem(cls, subsidy_uuid, learner_id, content_key):
+        """Subsidy service api"""
+        return mock.MagicMock(subsidy_uuid, learner_id, content_key)
+
+    @classmethod
+    def request_redemption(cls, subsidy_uuid, learner_id, content_key):
+        """Subsidy service api"""
+        return mock.MagicMock(subsidy_uuid, learner_id, content_key)
+
+    @classmethod
+    def has_redeemed(cls, subsidy_uuid, learner_id, content_key):
+        """Subsidy service api"""
+        return mock.MagicMock(subsidy_uuid, learner_id, content_key)
+
+    @classmethod
+    def has_requested(cls, subsidy_uuid, learner_id, content_key):
+        """Subsidy service api"""
+        return mock.MagicMock(subsidy_uuid, learner_id, content_key)
+
+    @classmethod
+    def get_license_for_learner(cls, subsidy_uuid, learner_id):
```

Comment:
nit: Please rename `license` to `subsidy` in this PR.

Reply:
@iloveagent57 should we change this name?

Reply:
@moconnell1453 @iloveagent57 Can we create the [access policy repo](https://github.com/openedx/edx-enterprise-subsidy-access-policy) also?